File: 104_鏈結串列/LinkedList/Node.java

public class Node
{
	int[] score;
	Node link;

	public Node(int num, int eng, int math)
	{
		score = new int[3];
		score[0] = num;
		score[1] = eng;
		score[2] = math;
		this.link = null;
	}
	public void setLink(Node link)
	{
		this.link = link;
	}
	public Node getLink()
	{
		return this.link;
	}
	public int[] getData()
	{
		return score;
	}
}

File: 201_圖形/TestGraph/Node.cs
using System;
namespace lesson201_01
{
    public class Node
    {
        private int data;
        private Node link;
        public Node(int data)
        {
            this.data = data;
            this.link = null;
        }
        public int GetData()
        {
            return this.data;
        }
        public Node GetLink()
        {
            return this.link;
        }
        public void SetLink(Node link)
        {
            this.link = link;
        }
    }
}
